docs/API_Reference.md

```diff
@@ -20,13 +20,14 @@ Currently, the eHSM-KMS-Service now provides the following restful APIs to the c
 
 ## Common Prameters
 This section describes the parameters that are common to all API requests and responses.
+
  | Name | Type | Reference Value | Description |
  |:-----------|:-----------|:-----------|:-----------|
  | appid | string | 12345678-0123-4567-*** | An unique id to request ehsm in a domain, which is requested<br> from ehsm service maintainer |
  | nonce | string  | 2866307081280200*** | 16 bytes random number, 15 minute validity; <br>Taking the current time as the benchmark, detect nonce within 15 minutes, which cannot be repeated  |
  | timestamp | string  | 20211215*** | The timestamp of sending request; format: yyyymmddhhmmssmmm |
```

Comment:
here nonce can be removed.  timestamp should be changed the wordings.

Reply:
It will be change.